tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "experimentalDecorators": true,
     "allowJs": true,
     "lib": ["es6", "dom"],
-    "rootDir": "./src",
+    "rootDir": "",
```

Comment:
All source files should be present in the rootDir. However, all the typescript components are failling the build process. There will be more changes coming to fix this.